src/power_grid_model_io/converters/base_converter.py

```diff
@@ -25,7 +25,7 @@ def __init__(
         self,
         source: Optional[BaseDataStore[T]] = None,
         destination: Optional[BaseDataStore[T]] = None,
-        log_level: int = logging.WARNING,
+        log_level: int = logging.CRITICAL,
```

Comment:
why? `WARNING` sounds like a good default. If GOAT doesn't agree, the should suppress our warnings to a different level, and we should go over which ones actually are spurious and which aren't

Reply:
it is actually the same from our perspective: if `pandapower` and `pandas` provide too many spurious warnings, we should catch those warnings on our end

Reply:
Processed in the latest commit, conversation resolved.